search/lib/stac/catalog.js

```diff
@@ -25,6 +25,12 @@ class Catalog {
     const child = link.createChild(title, `${root.href}${relativeUrl}`);
     this.links.push(child);
   }
+
+  addNext (title, relativeUrl) {
+    const root = this.links.find((l) => l.rel === this.links.RELATION_TYPES.child);
```

Comment:
I'm a bit curious why this is using `this.links.RELATION_TYPES.child` here instead of the `link.RELATION_TYPES.self` link for the root. Doesn't the href for the pagination link essentially go `${self.href}/page/:page_number`?

Reply:
I was going back and forth with this, ultimately I believe you're correct and have changed it to `link.RELATION_TYPES.self`.